lib/cassandra/version.rb

```diff
@@ -17,5 +17,5 @@
 #++
 
 module Cassandra
-  VERSION = '3.2.5'.freeze
+  VERSION = '3.2.6'.freeze
```

Comment:
```suggestion
  ##
  # ART19 maintains a fork with patches applied on top of the upstream gem.
  # We publish our fork with a revision number appended to the upstream version.
  #
  # @return [String] the ART19 revision number
  ART19_REVISION = '1'.freeze
  VERSION = "3.2.5.#{ART19_REVISION}".freeze
```

Reply:
ART19 has been doing a revision number approach to distinguish our patched gem versions from the upstream version